pkg/agent/cniserver/server_test.go

```diff
@@ -423,7 +429,7 @@ func translateRawPrevResult(prevResult *current.Result, cniVersion string) (map[
 	return conf.RawPrevResult, nil
 }
 
-func generateCNIServer(t *testing.T) *CNIServer {
+func constructCNIServer(t *testing.T) *CNIServer {
```

Comment:
How about `newCNIServer` to keep same style constructor name?

Reply:
Ok to me.